src/pro/javacard/gp/ArgMatchers.java

```diff
@@ -36,6 +36,32 @@ public AID convert(String arg0) {
 		}
 	}
 
+	public static ValueConverter<Hash> hash() {
```

Comment:
The whole ArgMatchers thingy in GPPro is not something to be satisfied with. Any ideas how to make it smaller-shorter-moreflexible are welcome.

Reply:
Giving it some thought, it might be nice to re-use Hex here and check for necessary size afterwards, to not introduce a somewhat dummy and single use class.